src/com/intellij/idea/plugin/hybris/toolwindow/RemoteHacConnectionDialog.kt

```diff
@@ -144,6 +158,60 @@ class RemoteHacConnectionDialog(
                     .onChanged { urlPreviewLabel.text = generateUrl() }
                     .component
             }.layout(RowLayout.PARENT_GRID)
+            if (System.getProperty("os.name").lowercase(Locale.getDefault()).contains("win")) {
```

Comment:
due fact that this logic is duplicated in both dialogs it can be moved to abstract class as a function

Reply:
Completed